src_features/signTx/logic_signTx.c

```diff
@@ -229,7 +229,7 @@ void computeFees(char *displayBuffer, uint32_t displayBufferSize) {
 }
 
 void finalizeParsing(bool direct) {
-    char displayBuffer[50];
+    char displayBuffer[79];  // required to store the string representation of uint256_t max
```

Comment:
let's take a more memory friendly approach: there's 112 million ETH, let's assume it inflates to 1 billion to have a realistic worst case scenario: the biggest ETH amount one could potentially spend has a length of 10 digits + '.' + 18 decimals digits + '\0' == 30 bytes

Reply:
Hmm so that stands for the `AMOUNT` field, but not for the `nonce` field. One could create an _almost_ infinite amount of tx (this might if Ethereum gets to a point where TPS is very high and there are connected device spamming the network...).
Admittedly though, 79 would mean that one has to do A LOT of transactions hahaha. I'm fine changing this back to 50, just wanted to make sure that this was OK with the reviewers :)

Reply:
I'm sure we'll boil the oceans before someone reaches a nonce with 50 digits, we can safely go back to 50 bytes